front/app/components/ProjectAndFolderCards/index.tsx

```diff
@@ -1,21 +1,32 @@
-import React, { useState, useEffect, useMemo, useCallback } from 'react';
+import React, { useState, useEffect, useMemo } from 'react';
 
 // hooks
 import useAdminPublicationsStatusCount from 'hooks/useAdminPublicationsStatusCounts';
 
 // components
-import ProjectAndFolderCards, { BaseProps } from './ProjectAndFolderCards';
+import ProjectAndFolderCardsInner from './ProjectAndFolderCardsInner';
 
 // utils
 import { isNilOrError } from 'utils/helperUtils';
 import { getCurrentTab } from './utils';
 
 // typings
 import { PublicationStatus } from 'services/projects';
+import { InputProps as UseAdminPublicationInputProps } from 'hooks/useAdminPublications';
 
 export type PublicationTab = PublicationStatus | 'all';
 
-export default ({ publicationStatusFilter, ...otherProps }: BaseProps) => {
+export type TLayout = 'dynamic' | 'threecolumns' | 'twocolumns';
+
+export interface BaseProps extends UseAdminPublicationInputProps {
+  showTitle: boolean;
+  layout: TLayout;
+}
```

Comment:
I also changed this in my PR. There's only 1 prop of UseAdminPublicationInputProps on ProjectFolderCards (the pub status filter) so wouldn't extend this type. It's clearer if you just add the 1 prop it's taking.